src/adaPets.js

```diff
@@ -19,19 +26,41 @@ const showDetails = (selectedPetId) => {
   }
 
   // Fill out as part of Wave 2.
+
+  axios.get(BASE_URL+selectedPetId)
+  .then((response)=>{
+    setResult(response.data);
+  })
+  .catch((error)=>{
+    setError(`There is an error when you tried show details or select error:${error}`);
+  });
+
 };
 
 const removePet = (selectedPetId) => {
   if (!selectedPetId) {
     setError("You tried to remove a pet without selecting it!");
     
   }
-
   // Fill out as part of Wave 3.
+  axios.delete(BASE_URL+selectedPetId)
+    .then((response)=>{
+      setResult(console.log(`pet id:${selectedPetId} is removed!`));
```

Comment:
You forgot to remove a `console.log`:

```suggestion
      setResult(`pet id:${selectedPetId} is removed!`);
```